chanzuckerberg/idseq-web
# Convert GET bulk download endpoints to POST.

# Description

These endpoints receive each sample id the user selects for bulk download. If the user selects many samples, the length of the GET url can get too large.

Theoretically, the GET url shouldn't have a maximum length, but various browsers have limits. See https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

POST requests have no limit.

# Tests

* Update tests.
* Verify manually that bulk download modal behaves as before.
